### DISCUSSION TITLE
info=target.image.error status='image.not.found' fedora37

### DISCUSSION THREAD

**Discussion by shiver25** [slimtoolkit/slim#510] [General]
Hi there,

I try use docker-slim in Fedora37 Linux and i have problem with this.

`shiver@fedora ~ $ docker pull docker.io/library/python
Trying to pull docker.io/library/python:latest...
Getting image source signatures
Copying blob f8c6dc678081 done  
Copying blob 68a71c865a2c done  
Copying blob 5a7a2c95f0f8 done  
Copying blob 6d627e120214 done  
Copying blob 670730c27c2e done  
Copying blob 3e440a704568 done  
Copying blob 1a3aca716a48 done  
Copying blob 3874615e0f8a done  
Copying blob 1b2574f1d1ea done  
Copying config 148bdd2c54 done  
Writing manifest to image destination
Storing signatures
`148bdd2c547fb9a80acac8cd1fca27870d840874d132420795324e3ab2992597`

`shiver@fedora ~ $ docker-slim build docker.io/library/python:latest
docker-slim: message='join the Gitter channel to ask questions or to share your feedback' info='https://gitter.im/docker-slim/community'
docker-slim: message='join the Discord server to ask questions or to share your feedback' info='https://discord.gg/9tDyxYS'
docker-slim: message='Github discussions' info='https://github.com/docker-slim/docker-slim/discussions'
cmd=build info=param.http.probe message='using default probe' 
cmd=build state=started
cmd=build info=params keep.perms='true' tags='' target='docker.io/library/python:latest' continue.mode='probe' rt.as.user='true' 
cmd=build info=target.image.error status='image.not.found' image='docker.io/library/python:latest' message='make sure the target image already exists locally (use --pull flag to auto-download it from registry)' 
cmd=build state=exited code=33554435
docker-slim: message='join the Gitter channel to ask questions or to share your feedback' info='https://gitter.im/docker-slim/community'
docker-slim: message='join the Discord server to ask questions or to share your feedback' info='https://discord.gg/9tDyxYS'
docker-slim: message='Github discussions' info='https://github.com/docker-slim/docker-slim/discussions'
`

**Comment by kcq** [comment]
This is odd... Slim complains that it can't find the image, but you pull the python image right above. Need to repro it with Fedora. There might be some kind of environmental gotcha.
